openmdao/drivers/scipy_optimizer.py

```diff
@@ -216,6 +220,10 @@ def run(self):
         # maxiter and disp get passsed into scipy with all the other options.
         self.opt_settings['maxiter'] = self.options['maxiter']
         self.opt_settings['disp'] = self.options['disp']
+        if opt == 'COBYLA':
+            self.opt_settings['rhobeg'] = self.options['initstep']
+            self.opt_settings['catol'] = self.options['catol_cob']
```

Comment:
The formatting is bad here, delete one of the blank lines.

Reply:
thanks. i corrected this. I cant seem to be able to assign reviewers. (which I assume, has to be done by me not by the developers?)